src/library_wasmfs_opfs.js

```diff
@@ -132,7 +132,11 @@ mergeInto(LibraryManager.library, {
   _wasmfs_opfs_get_entries__deps: [],
   _wasmfs_opfs_get_entries: async function(ctx, dirID, entries) {
     let dirHandle = wasmfsOPFSDirectories.get(dirID);
-    for await (let [name, child] of dirHandle.entries()) {
+
+    // TODO: Use 'for await' once Acorn supports that.
+    let iter = dirHandle.entries();
+    for (let entry; entry = await iter.next(), !entry.done;) {
+      let [name, child] = entry.value;
```

Comment:
We also have yet to stat using `for .. of` in the emscripten codebase because we don't yet have a good way to polyfil it on older browers.

Perhaps its true that one would never ship the OPFS code to such browsers.. but maybe you could imagine an OPFS-using app that falls back to not using it when it not available?  i.e. on older browsers.